lib/Migration/Version140000Date202503027114917.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\TwoFactorTOTP\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\QueryBuilder\IQueryBuilder;
+use OCP\IDBConnection;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version140000Date202503027114917 extends SimpleMigrationStep {
+	public function __construct(
+		private IDBConnection $db,
+	) {
+	}
+
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @psalm-param Closure():ISchemaWrapper $schemaClosure
+	 * @param array $options
+	 * @since 13.0.0
+	 */
+	public function postSchemaChange(IOutput $output, \Closure $schemaClosure, array $options) {
+		$qb = $this->db->getQueryBuilder();
+		$qb->select('*')->from('twofactor_totp_secrets')->setMaxResults(1);
+		$result = $qb->executeQuery();
+		$row = $result->fetch(\PDO::FETCH_ASSOC);
+
+		if (!isset($row['verified'])) {
+			$result->closeCursor();
+			return;
+		}
```

Comment:
nit: might be able to also use the schema closure and ask if the table column exists

Reply:
Mh, not sure how that would look like

Reply:
```php
		$schema = $schemaClosure();
		$table = $schema->getTable('twofactor_totp_secrets');
		if ($table->hasColumn('verified')) {
			// ...
		}
```

Reply:
I was wondering whether I can execute SQL in the changeSchema method, but I can just set an attribute.